docs/source/tools/ledger-api-test-tool/index.rst

```diff
@@ -160,3 +160,20 @@ will still print information about failed tests.
 
 We used this flag during tool development to ensure that the tool does not
 always return success.
+
+Tuning the testing behaviour of the tool
+========================================
+
+Use the command line options ``--timeout-scale-factor`` and
+``--command-submission-ttl-scale-factor`` to tune timeouts applied by the tool:
+
+- set ``--timeout-scale-factor`` to a floating point value higher than 1.0 to make
+  the tool wait longer for expected events coming from the DAML ledger
+  implementation under test. Conversely use values smaller than 1.0 to make it
+  wait shorter;
```

Comment:
```suggestion
  wait shorter.
```

Sorry, missed this the first time around.

Reply:
I've also updated the capitalization of the first word of the sentences in the bullet points.

Reply:
Thanks.